Day-I-Social-Card/social-card/src/App.css

```diff
@@ -2,3 +2,20 @@
   Write your styles/CSS for App.js here and import into the component that you need 
   IF you'd like to attempt to use less you'll have to research how to do this with Create React App
 */
+.cardContainer {
```

Comment:
Don't forget about `box-sizing: border-box` and constraints (width and max-width). It will make your CSS life much easier. A good place for those could be index.css but it doesn't really matter if you put it in app.css or index.css.